cli/dcos-spark/submit_builder.go

```diff
@@ -221,6 +220,12 @@ Args:
 	val.flag(submit).StringVar(&val.s)
 	args.stringVals = append(args.stringVals, val)
 
+	val = newSparkVal("packages", "spark.jars.packages", "Comma-separated list of maven coordinates of jars to include "+
+                "on the driver and executor classpaths. Will search the local maven repo, then maven central and any additional remote "+
+                "repositories given by --repositories. The format for the coordinates should be groupId:artifactId:version")
+        val.flag(submit).StringVar(&val.s)
+        args.stringVals = append(args.stringVals, val)
```

Comment:
we need to add `--conf spark.jars.ivy` so that users don't worry about it

Reply:
Added `--conf spark.jars.ivy` while processing `--packages` flag in the CLI.